### DISCUSSION TITLE
Overlays Meeting (2024-11-05)

### DISCUSSION THREAD

**Discussion by ralfhandl** [OAI/Overlay-Specification#90] [Meeting Notes]
Housekeeping
* #64 
* #92 
* #93 

Schema development
* #86

**Comment by lornajane** [comment]
Upcoming meeting platform change

**Comment by mikeschinkel** [comment]
Discussed on the call: [#108 — Change name of component schema item, delete schema item properties, create subset based on of a schema ](https://github.com/OAI/Overlay-Specification/discussions/108).
